portfolio/src/main/webapp/index.html

```diff
@@ -12,55 +12,99 @@
   </head>
   <body>
     <div id="content">
-        
+
+        <!-- main page -->
 
         <div class="main" id="home">
-            <img src="images/name.gif" width = "600" class="responsive center">
-            <img src="images/Ariel.gif" width = "600" class="responsive center">
+            <div class="flexbox">
+
+            <article class="mid">
+                <div class="mainContainer center">
+                    <br><br><br></br>
```

Comment:
This would be better done with a margin CSS setting.

Reply:
Changed and committed this morning! However, it incorrectly didn't go through the PR-review process, and is already merged.

Reply:
> This would be better done with a margin CSS setting.

Changed and committed this morning! However, it incorrectly didn't go through the PR-review process, and is already merged.